lib/ansible/modules/windows/win_domain_computer.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright 2017, AMTEGA - Xunta de Galicia
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# this is a windows documentation stub.  actual code lives in the .ps1
+# file of the same name
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = r'''
+---
+module: win_domain_computer
+short_description: Management of Computer Active Directory Objects
+description:
+  - >
+    Create, read, update and delete computers in Active Directory using a
+    windows brigde computer to launch New-ADComputer, Get-ADComputer,
+    Set-ADComputer, Remove-ADComputer and Move-ADObject.
+options:
+  name:
+    description:
+      - >
+        Specifies the name of the object. This parameter sets the Name property
+        of the Active Directory object. The LDAP display name (ldapDisplayName)
+        of this property is name.
+    required: true
+  sam_account_name:
+    description:
+      - >
+        Specifies the Security Account Manager (SAM) account name of the user,
+        group, computer, or service account. The maximum length of the
+        description is 256 characters. To be compatible with older operating
+        systems, create a SAM account name that is 15 characters or less. This
+        parameter sets the SAMAccountName for an account object. The LDAP
+        display name (ldapDisplayName) for this property is sAMAccountName.
+        Note: If the SAMAccountName string provided does not end with a $, a $
```

Comment:
I would keep this straightforward, either ask for it explicitly, or add it yourself always. We don't need magic and we don't need examples on the net doing it differently because it adds to the confusion.

Maybe check what other modules do for the same value ?

Reply:
Also, it would be better of the description is split up over multiple items. A long blob doesn't make it easier to read.

Usually the first item is the "meaning", and subsequent items detail what happens in different cases, explain options, specify the format or add one or more notes.

Reply:
☑️ 
Hope it makes more sense